src/data/defaults.txt

```diff
@@ -3474,4 +3478,4 @@ user	lastNoncombat450	0
 user	lastNoncombat533	0
 user	lastNoncombat539	0
 user	lastNoncombat540	0
-user	lastNoncombat541	0
+user	lastNoncombat541	0
```

Comment:
What changed?  I assume whitespace but since we have tests that should catch some whitespace errors in data files it would be useful to know whether the tests need to be looked at.

Reply:
I took another look at defaults.txt - it looks like the original has an extra newline at the end and my version does not

Reply:
And fixed